07LessonSeven/in-class-project/index.html

```diff
@@ -3,8 +3,58 @@
 
 <head>
     <title>Template</title>
+    <link type="text/css" rel="stylesheet" href="css/my-style.css">
+
 </head>
 
-<body></body>
+<body>
+
+      <div class="image">
+         <img class="image" src="img/nasa-background.jpg" alt="" />
+         <p class="text"><strong>NASA</strong></p>
+      </div>
+

```

Comment:
the nasa container looks great! that's one way to get the text in front of the image. Consider trying out the other way in your css by using the background-image property though. having position: absolute can be a hassle at times.